### PR TITLE
Update /iot page with smart start changes

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done
**This is part of a round of smart start related updates that are all to go live at the same time**

- Updated /internet-of-things to match the [copy doc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the page matches the copy doc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5082

### CODE REVIEW

File: templates/internet-of-things/index.html [hunk 1]
```diff
@@ -191,11 +220,39 @@ <h2>IoT App Store</h2>
...
+        <li class="p-list__item is-ticked">Fast time to market, low risk, and scalable</li>
+      </ul>
+
+      <p>
+        <a href="/internet-of-things/contact-us?product=iot-overview" class="p-button--positive js-invoke-modal">Contact us</a>
+        <a href="https://ubuntu.com/pricing/devices">Available pricing and packages&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
```
Comment (minkyngkm) [inline templates/internet-of-things/index.html:241]:
```suggestion
    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
```

File: templates/internet-of-things/index.html [hunk 2]
```diff
@@ -167,11 +167,40 @@ <h3>Smart city</h3>
   </div>
 </section>
 
-<section class="p-strip--light is-bordered">
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
```
Comment (minkyngkm) [inline templates/internet-of-things/index.html:172]: This needs to be `col-6`?
```suggestion
    <div class="col-6 u-align--center u-vertically-center u-hide--medium u-hide--small">
```

**Comment by minkyngkm** [comment]
Oh and this sign `>` is missing on some links if it needs to be added.

File: templates/internet-of-things/index.html (hunk 1 above)
Comment (sowasred2012) [reply templates/internet-of-things/index.html:241]: :+1:

**Pull request merged**